sound/soc/sof/intel/hda-loader.c

```diff
@@ -25,6 +25,44 @@
 
 #define HDA_CL_STREAM_FORMAT 0x40
 
+static void hda_set_ssps_slave(struct snd_sof_dev *sdev)
```

Comment:
typos in commit message: "Move the SSP DAI's slave mode set function to the hda_set_ssps_slave() helper and the SDW wakeen interrupt enabling to the hda_sdw_wakeen_resume() helper"